File: Plugins/Aspose_Barcode_Java_for_PHP/src/aspose/barcode/WorkingWithBarcodeRecognition/BasicBarcodeRecognitionFeatures/RecognizeBarcode.php
<?php
namespace Aspose\Barcode\WorkingWithBarcodeRecognition\BasicBarcodeRecognitionFeatures;

use com\aspose\barcoderecognition\BarCodeReadType as BarCodeReadType;
use com\aspose\barcoderecognition\BarCodeReader as BarCodeReader;

class RecognizeBarcode{

    public static function run($dataDir=null){

        # initialize barcode reader
        $img = $dataDir . "barcode.jpg";
        $barcode_reader_type = new BarCodeReadType();

        $rd = new BarCodeReader($img, $barcode_reader_type->Code39Standard);

        # read barcode
        while (java_values($rd->read())) {
            # print the code text, if barcode found
            print "CodeText: " . (string)$rd ->getCodeText().PHP_EOL;

            # print the symbology type
            print "Type: " . (string)$rd ->getReadType().PHP_EOL;
            break;
        }
    }
}

File: Plugins/Aspose-BarCode-Java-for-Jython/asposebarcode/WorkingWithBarcodeRecognition/BasicBarcodeRecognitionFeatures/RecognizeBarcode.py
from asposebarcode import Settings
from com.aspose.barcoderecognition import BarCodeReadType
from com.aspose.barcoderecognition import BarCodeReader

class RecognizeBarcode:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithBarcodeRecognition/AdvancedBarcodeRecognitionFeatures/RecognizeBarcode/'
        
       # initialize barcode reader
        img = dataDir + "barcode.jpg"
        barcode_reader_type = BarCodeReadType

        rd = BarCodeReader(img, barcode_reader_type.Code39Standard)

        # read barcode
        while (rd.read()):
            # print the code text, if barcode found
            print "CodeText: " 
            print rd.getCodeText()

            # print the symbology type
            print "Type: "
            print rd.getReadType()
            break

if __name__ == '__main__':        
    RecognizeBarcode()